packages/x-date-pickers-pro/src/SingleInputTimeRangeField/SingleInputTimeRangeField.tsx

```diff
@@ -5,13 +5,15 @@ import { useClearableField } from '@mui/x-date-pickers/hooks';
 import { convertFieldResponseIntoMuiTextFieldProps } from '@mui/x-date-pickers/internals';
 import { PickerValidDate } from '@mui/x-date-pickers/models';
 import { PickersTextField } from '@mui/x-date-pickers/PickersTextField';
-import { useThemeProps } from '@mui/material/styles';
 import { useSlotProps } from '@mui/base/utils';
 import { refType } from '@mui/utils';
+import { createUseThemeProps } from '../internals/zero-styled';
 import { SingleInputTimeRangeFieldProps } from './SingleInputTimeRangeField.types';
 import { useSingleInputTimeRangeField } from './useSingleInputTimeRangeField';
 import { FieldType } from '../models';
 
+const useThemeProps = createUseThemeProps('MuiSingleInputTimeRangeField');
```

Comment:
Honestly I have no idea
I think I read it month ago. I'd say it's just that PigmentCSS will replace it be an import links it to the build-time theme based on the name that is in `createUseThemeProps` and ignore the one of `useThemeProps` and we kept the one in `useThemeProps` to avoid a breaking change.